cmd/tuf/app/sign.go

```diff
@@ -55,7 +56,7 @@ func Sign() *ffcli.Command {
 		roles      = roleFlag{}
 		repository = flagset.String("repository", "", "path to the staged repository")
 		sk         = flagset.Bool("sk", false, "indicates use of a hardware key for signing")
-		key        = flagset.String("key", "", "reference to an onine signer for signing")
+		key        = flagset.String("key", "", "reference to an online signer for signing")
```

Comment:
is this incorrect now? online could also mean on-disk

Reply:
oh! I meant maybe just remove the online word

Reply:
Yeah, that's maybe better. I'll do it.